ga4gh/server/network/__init__.py

```diff
@@ -0,0 +1,76 @@
+"""
+Provides methods to initialize the server's peer to peer connections.
+
+"""
+from __future__ import division
+from __future__ import print_function
+from __future__ import unicode_literals
+
+import os
+import requests
+
+from ga4gh.server import datamodel, exceptions
```

Comment:
I had you in mind when I wrote that line @dcolligan :). I do like that we have uniform import statements, I just find this syntax nicer to read. I'll change it back but continue to try to convince you that we could clean up the code base by switching to this style.

```
import ga4gh.server.datamodel.datasets as datasets
import ga4gh.server.datamodel.ontologies as ontologies
import ga4gh.server.datamodel.reads as reads
import ga4gh.server.datamodel.references as references
import ga4gh.server.datamodel.variants as variants
import ga4gh.server.datamodel.sequence_annotations as sequence_annotations
import ga4gh.server.datamodel.continuous as continuous
import ga4gh.server.datamodel.bio_metadata as biodata
import ga4gh.server.datamodel.genotype_phenotype as genotype_phenotype
import ga4gh.server.datamodel.genotype_phenotype_featureset as g2pFeatureset
import ga4gh.server.datamodel.rna_quantification as rna_quantification
```

Might become:

```

from ga4gh.server.datamodel import ontologies, reads, references, variants, \
    sequence_annotations, continuous,  biodata, genotype_phenotype, rna_quantification
```

I think it might actually be more PEPpy?

http://stackoverflow.com/questions/15011367/python-module-import-single-line-vs-multi-line